scene/gui/tabs.cpp

```diff
@@ -305,7 +308,8 @@ void Tabs::_notification(int p_what) {
 			Ref<Texture2D> incr_hl = get_theme_icon("increment_highlight");
 			Ref<Texture2D> decr_hl = get_theme_icon("decrement_highlight");
 
-			int limit = get_size().width - incr->get_size().width - decr->get_size().width;
+			int limit = get_size().width;
+			int limit_minus_buttons = get_size().width - incr->get_size().width - decr->get_size().width;
```

Comment:
You can change `get_size().width` to `get_width()`.

Reply:
Done :)